src/main/scala/diplomacy/Nodes.scala

```diff
@@ -49,10 +133,21 @@ trait OutwardNodeImp[DO, UO, EO, BO <: Data]
   def getI(pd: DO): Option[BaseNode] = None // most-inward common node
 }
 
+/**
+ * A node implementation.
+ *
+ * This class has no members and is solely used for holding type information.
+ * Applications of Diplomacy should extend NodeImp with a case object that sets
+ * concrete type arguments.
+ */
```

Comment:
```suggestion
/**
  * A node implementation.
  *
  * This class has no members and is solely used for holding type information.
  * Applications of Diplomacy should extend NodeImp with a case object that sets
  * concrete type arguments.
  */
```
This applies to every one of these, if the following starts line up with the left `*` it is JavaDoc, you line up with the right `*` for ScalaDoc

Reply:
Looks like I need to go figure out how to change this in my vim settings. It was automatically inserting the `*` on each new line, aligned to the left one.

...And it looks like it's right there at the [bottom of the `vim-scala` README](https://github.com/derekwyatt/vim-scala#scaladoc-comment-indentation). Kind of annoying that it's not on by default.

Reply:
Fixed my editor settings in https://github.com/richardxia/dotfiles/commit/55518fbd1166ca7fde2266fabfdfcd14e831bfdf.

I'll remember to fix up all of these later.